src/components/About/Blog/index.js

```diff
@@ -137,9 +141,15 @@ const BlogNavigation = styled.div`
 `;
 
 const SocialButtons = styled.div`
-  width: 100%;
+  width: 30%;
   display: flex;
   padding: 0.625rem 0 1.25rem 0.625rem;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
Do not use `space-between` here. Use `flex-start` and let the width be `100%`. The spacing is excessive in your current implementation.